app/views/includes/degateway-banner.njk

```diff
@@ -0,0 +1,9 @@
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{{ govukPhaseBanner({
+  attributes: {id: "degateway-phase-banner"},
+  tag: {
+    text: "Experimental",
+    classes: "govuk-tag--orange"
+  },
+  html: 'This is a test version of this page, intended only for use by the GOV.UK Pay team in GDS. You currently have \'Degatewayaccountification\' enabled in \'My profile\'.<br />If you think you should not be seeing this, please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact the GOV.UK Pay Team</a>.'
```

Comment:
this banner will show when the user is 'degatewayed' even if the ability to see the setting in 'My Profile' is disabled. 

this could end up being confusing

Reply:
That's an intentional decision - it allows us to each create an account opted in to this functionality, then turn off the option so that test more closely resembles prod for all other users.